tests/structured_conf/data/attr_classes.py

```diff
@@ -221,6 +221,20 @@ class StringOptional:
     not_optional: str = "foo"
 
 
+@attr.s(auto_attribs=True)
+class ListOptional:
+    with_default: Optional[List[int]] = [1, 2, 3]
+    as_none: Optional[List[int]] = None
+    not_optional: List[int] = [1, 2, 3]
+
+
+@attr.s(auto_attribs=True)
+class TupleOptional:
```

Comment:
Since Tuple was not supported before, also add non optional tests for Tuple.

Reply:
> I think you missed adding an actual test that interacts with your new ListOptional and TupleOptional classes.

I added those two classes on 'test_optional'